src/cli/cli.cpp

```diff
@@ -1453,6 +1453,20 @@ template <> otError Interpreter::Process<Cmd("delaytimermin")>(Arg aArgs[])
 }
 #endif
 
+template <> otError Interpreter::Process<Cmd("detachgracefully")>(Arg aArgs[])
```

Comment:
```suggestion
template <> otError Interpreter::Process<Cmd("detach")>(Arg aArgs[])
```

Better use `gracefully` as an argument?

`> detach`: detaches from Thread network immediately and enters disabled state.
`> detach gracefully`: detaches from Thread network gracefully and enters disabled state afterwards.

Reply:
SGTM.

Reply:
Done.